modules/media/php/edit.class.inc

```diff
@@ -41,10 +33,10 @@ class NDB_Form_Media extends NDB_Form
      *
      * @return void
      */
-    function edit()
+    function _setupPage()
```

Comment:
`Declaration of LORIS\\media\\Edit::_setupPage() should be compatible with NDB_Page::_setupPage($name, $page, $identifier, $commentID, $formname)`

Reply:
There should also be a parent call here.

Reply:
This was not triggering error in the manual test because there is not `Edit Metadate` link in the table. This is due to the fact that `$this->hasWritePermission` is always false in `modules/media/php/media.class.inc` because the `_hasAccess` function is called after `_setupPage`. PR on the way...